src/applications/appeals/10182/config/form.js

```diff
@@ -91,12 +91,16 @@ const formConfig = {
   confirmation: ConfirmationPage,
 
   defaultDefinitions: {},
+  // when true, initial focus on page to H3s by default, and enable page
+  // scrollAndFocusTarget (selector string or function to scroll & focus)
+  useCustomScrollAndFocus: true,
+
   chapters: {
     infoPages: {
-      title: 'Veteran details',
+      title: 'Veteran Information',
       pages: {
         veteranInformation: {
-          title: 'Veteran details',
+          title: 'Veteran information',
           path: 'veteran-details',
```

Comment:
do we want to update path to `veteran-information` for consistency?

Reply:
No, because it would break the save-in-progress for existing Veterans. I'd have to include a `migration` to get around that - that's out of scope for this work. I probably didn't need to update this since this title isn't seen anywhere.